clickhouse_driver/client.py

```diff
@@ -376,11 +377,11 @@ def execute_iter(
         """
 
         with self.disconnect_on_error(query, settings):
-            return self.iter_process_ordinary_query(
+            return chunks(self.iter_process_ordinary_query(
```

Comment:
If chunk_size is 1 it should be no additional overhead

```python
rv = self.iter_process_ordinary_query(...)

return chunks(rv, chunk_size) if chunk_size > 1 else rv
```

Reply:
Yeah, it's much better.